.changeset/cool-countries-pay.md

```diff
@@ -0,0 +1,24 @@
+---
+'@remirror/core': minor
```

Comment:
Would be awesome if you could also add every extension that is affected by the changes.